samples/sample-client-common/src/main/java/com/flipkart/aesop/sample/client/common/events/MysqlBinLogEventImpl.java

```diff
@@ -127,4 +136,67 @@ public String toString()
 		return builder.toString();
 	}
 
+	/**
+	 * Check whether given field is rowChangeField
+	 * @param field
+	 * @return True/False
+	 */
+	private boolean isRowChangeField(Schema.Field field)
+	{
+		String fieldValue = SchemaHelper.getMetaField(field, META_ROW_CHANGE_FIELD);
+		return fieldValue != null;
+	}
+
+	/**
+	 * Generates a hash storing fieldName to intended MysqlDataType from schema
+	 * @param schema
+	 * @return Map containg FieldName to Mysql Type mapping
+	 */
+	private Map<String, String> fieldToDataTypeMap(Schema schema)
```

Comment:
Map<String, String> map = new HashMap<>(schema.getFields().size()) (or some equivalent which specifies the hash map size in the constructor itself)

Also, remove the unnecessary mysqlType declaration.